src/static/js/modules/form-explainer.js

```diff
@@ -286,7 +274,18 @@ $(document).ready(function(){
       duration: 200,
       offset: -30
     });
-    $( itemID ).get(0).toggle();
+    $( itemID )
+      .attr('tabindex', '0').attr('aria-hidden', 'false')
+      .get(0).toggle();
+
+  });
+
+  // Add tabindex to links inside of expandables, so they are tabbable in the correct order.
+  $WRAPPER.on( 'focus', '.expandable__form-explainer .expandable_target', function (event) {
+    var $this = $(this),
+        $content = $this.siblings('.expandable_content'),
+        $tabindex = $content.attr('tabindex');
+    $content.find('a').attr('tabindex', $tabindex);
   });
```

Comment:
Could we iterate through all the expandables and add tabindex values to their links on page load?